pages/deployment/[uuid].tsx

```diff
@@ -128,18 +134,18 @@ export default function Deployment() {
   if (error) return error
 
   if (isDeploymentLoading || isUiConfigLoading || isCurrentUserLoading) {
-    return <Wrapper title={'Loading...'} page={'deployment'} />
+    return <Wrapper title='Loading...' page='deployment' />
   }
 
-  const deploymentTag = `${uiConfig?.registry.host}/${currentUser!.id}/${tag}`
+  const deploymentTag = `${uiConfig?.registry.host}/${currentUser?.id}/${tag}`
```

Comment:
This change isn't valid.  If `currentUser` is undefined we should error, instead of returning `undefined` here.

Reply:
I note that at this stage `currentUser` cannot be undefined.  So, instead we should use a `Type Guard` to tell TypeScript that it's not undefined.